src/ledger/test/LedgerCloseMetaStreamTests.cpp

```diff
@@ -0,0 +1,145 @@
+// Copyright 2019 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "crypto/Hex.h"
+#include "crypto/Random.h"
+#include "history/HistoryArchiveManager.h"
+#include "history/test/HistoryTestsUtils.h"
+#include "lib/catch.hpp"
+#include "main/Application.h"
+#include "main/ApplicationUtils.h"
+#include "test/TestUtils.h"
+#include "test/test.h"
+#include "xdr/Stellar-ledger.h"
+
+using namespace stellar;
+
+#ifndef _WIN32
```

Comment:
Is there a reason that those tests are not written cross platform?

Reply:
I initially thought I'd be using anonymous `pipe()` rather than a file-backed fd. I suppose the file-backed fd should work on windows too. I'll give it a try.

Reply:
windows tests work now too. and confirmed named pipes still work.